code/datums/gas_mixture.dm

```diff
@@ -460,83 +448,44 @@ What are the archived variables for?
 	toxins -= delta_toxins
 	sharer.toxins += delta_toxins
 
-	var/moved_moles = (delta_oxygen + delta_carbon_dioxide + delta_nitrogen + delta_toxins)
-	last_share = abs(delta_oxygen) + abs(delta_carbon_dioxide) + abs(delta_nitrogen) + abs(delta_toxins)
```

Comment:
`last_share = abs(delta_oxygen) + abs(delta_carbon_dioxide) + abs(delta_nitrogen) + abs(delta_toxins)` this needs to stay.